enocean/README.md

```diff
@@ -342,40 +418,50 @@ EnOcean_Item:
             enocean_rx_key = VALUE
     
     [[actor1B]]
-        enocean_rx_id = FFAABBCD
+        remark = Eltako FSR61, FSR61NP, FSR61G, FSR61LN, FLC61NP - Switch for Ligths
+        enocean_rx_id = 1A794D3
         enocean_rx_eep = F6_02_03
         [[[light]]]
             type = bool
-            enocean_rx_key = B
             enocean_tx_eep = A5_38_08_01
-            enocean_tx_id_offset = 2
+            enocean_tx_id_offset = 1
+            enocean_rx_key = B
+            block_switch = 'False'
+            cache = 'True'
+            enforce_updates = 'True'
+            visu_acl = rw
     
     [[actorD2]]
+        name: ???
+        remark = Actor with VLD Command
         enocean_rx_id = FFDB7381
         enocean_rx_eep = D2_01_07
         [[[move]]]
-            type=bool
+            type = bool
             enocean_rx_key = STAT
             enocean_tx_eep = D2_01_07
-            # optional; turn off after 0.1 s
-            enocean_pulsewidth = 0.1
-            enocean_tx_id_offset = 1  
+            enocean_tx_id_offset = 1
+            # pulsewith-attribute removed use autotimer functionality instead
+            autotimer = 1 = 0  
     
     [[awning]]
-        enocean_rx_id = 0500E508
-        enocean_rx_eep = F6_02_03
+        remark = Eltako FSB14, FSB61, FSB71 - actor for Shutter
+        type = str
+        enocean_rx_id = 1A869C3
+        enocean_rx_eep = F6_02_03_01
+        enocean_rx_key = STATUS
         [[[move]]]
-            enocean_tx_eep = A5_3F_7F
-            # move awning for 60 s
-            enocean_rtime = 60
             type = num
-            visu = yes
-        [[[stat]]]
+            enocean_tx_eep = A5_3F_7F
+            enocean_tx_id_offset = 0
             enocean_rx_key = B
-            enforce_updates = on
-            cache = on
-            type = bool
-            visu = yes
+            # runtime Range [0 - 255] s
+            enocean_rtime = 60
```

Comment:
Such a range limit could also be add as `min` and `max` value for an item setting. No need to describe it there.

Reply:
I have added the min and max function for the values to the calculations for the telegram data.
It is intended just for information for the user.
Should I though remove it?

Reply:
I was speaking from the `plugin.yaml` configuration. This can be used to a) describe parameters and b) to validate parameters based on type and restrictions (details see end of [documentation page](http://smarthomeng.de/developer/development_plugin/plugin_metadata.html)).